src/Domain.LinnApps/External/IRequisitionStoredProcedures.cs

```diff
@@ -30,5 +30,7 @@ Task<ProcessResult> PickStock(
             string transactionCode);
             
         Task<ProcessResult> CreateRequisitionLines(int reqNumber, int? serialNumber);
+
```

Comment:
you could expand the can book logic in the domain logic if it doesn't do what what you need as I decided not to use the STORES_OO logic but rewrite it

Reply:
ah ok didn't realise that